packages/preview/src/browser/style/markdown.css

```diff
@@ -0,0 +1,251 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
please add our copyright as well

Reply:
actually, I'm not sure, should we leave the theming switch based on `.vs` and `.vs-dark` as is, or rewrite with `theia`?

Reply:
theia based will be better if it is possible